CalcualtorLogic.js

```diff
@@ -79,30 +79,33 @@ class Calculator {
             buttonElement.appendChild(buttonElementText);
             div.appendChild(buttonElement);
             buttonElement.addEventListener('click', function() {
-                if (button === "Operator") result.textContent = "";
+                if (button === "Operator") { result.textContent = "" };
```

Comment:
Please don't create a new function here
just pass the `onButtonNumberClick` method like this

```js 
buttonElement.addEventListener("click", this.onButtonNumberClick); 

```
notice, that just passed the method. we didn't inovke the method 

in other words we didn't do it this way

```js 

buttonElement.addEventListener("click", this.onButtonNumberClick ()) ;


```